yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/AttemptsConfiguration.java

```diff
@@ -0,0 +1,75 @@
+package com.yoti.api.client.docs.session.create;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public final class AttemptsConfiguration {
+
+    @JsonProperty("ID_DOCUMENT_TEXT_DATA_EXTRACTION")
+    private final Map<String, Integer> idDocumentTextDataExtractionAttemptsConfig;
+
+    private AttemptsConfiguration(Map<String, Integer> idDocumentTextDataExtractionAttemptsConfig) {
+        this.idDocumentTextDataExtractionAttemptsConfig = idDocumentTextDataExtractionAttemptsConfig;
+    }
+
+    public static AttemptsConfiguration.Builder builder() {
+        return new AttemptsConfiguration.Builder();
+    }
+
+    /**
+     * Returns the attempts configuration {@link Map} for ID document text extraction task retries
+     *
+     * @return the attempts configuration map
+     */
+    public Map<String, Integer> getIdDocumentTextDataExtractionAttemptsConfig() {
+        return idDocumentTextDataExtractionAttemptsConfig;
+    }
+
+    public static final class Builder {
+
+        private static final String RECLASSIFICATION = "RECLASSIFICATION";
+        private static final String GENERIC = "GENERIC";
+
+        private Map<String, Integer> idDocumentTextDataExtractionAttempts;
+
+        private Builder() {}
+
+        private Builder withIdDocumentTextExtractionCategoryRetries(String category, int numberOfRetries) {
+            if (idDocumentTextDataExtractionAttempts == null) {
+                idDocumentTextDataExtractionAttempts = new HashMap<>();
+            }
```

Comment:
maybe move this to the constructor? Always initialize collections in the Builder constructor

Reply:
We don't want this though - that means we would always send an empty object when serialising to JSON.  If we add other property maps in here, we don't want to default the others to empty objects (rather we want to omit them).

Reply:
So u're saying that server side u want either no json property (null map) or not empty json property (map with values).

Reply:
Yes - it just ensures that the SDK is never opinionated, if the integrator doesn't call those methods for that specific property.